test/e2e/conditions.go

```diff
@@ -0,0 +1,61 @@
+package e2e
+
+import (
+	"time"
+
+	. "github.com/onsi/gomega"
+
+	"k8s.io/apimachinery/pkg/types"
+	yaml "sigs.k8s.io/yaml"
+
+	corev1 "k8s.io/api/core/v1"
+
+	nmstatev1alpha1 "github.com/nmstate/kubernetes-nmstate/pkg/apis/nmstate/v1alpha1"
+)
+
+type ExpectedConditionsStatus struct {
```

Comment:
Do we need this to be public?

BTW, consider creating a separate package for condition helpers, we should start making test utils more structured.